wsd/DocumentBroker.hpp

```diff
@@ -485,7 +485,14 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
             _status = newStatus;
         }
 
-        /// Flag close-requested. Cannot be reset.
+        /// True iff the document is fully loaded and available for viewing/editing.
+        bool isLive() const { return _status == Status::Live; }
```

Comment:
Not used anywhere, will that be used later?

Reply:
Right. The poll function will be simplified to check for specific states. isLive() is the steady-state, for most of the normal operation and !isLive() will be for the cases we do at startup and at shutdown of the document.